src/cloud-api-adaptor/Makefile

```diff
@@ -93,7 +95,7 @@ test: ## Run tests.
 .PHONY: test-e2e
 test-e2e: ## Run end-to-end tests for single provider.
 ifneq ($(CLOUD_PROVIDER),)
-	go test -v -tags=$(CLOUD_PROVIDER) -timeout $(TEST_E2E_TIMEOUT) -count=1 ./test/e2e
+	go test -v -tags=$(CLOUD_PROVIDER) -timeout $(TEST_E2E_TIMEOUT) -count=1 -run $(RUN_TESTS) ./test/e2e
```

Comment:
Nice one - I've been making this change exact change manually every so often for over a year, and keep meaning to push it, so thanks for not being as lazy as me!

Reply:
Every morning I say "I need to send this way of running just a sub-set of tests" and that passed years too. Today is a new era, today the cycle was broken! :)

Reply:
Unfortunately you have just demonstrated that if I put off an issue long enough it will go away!